src/pages/index.tsx

```diff
@@ -15,6 +16,7 @@ function HomepageHeader() {
       <div className="container">
         <h1 className="hero__title">{siteConfig.title}</h1>
         <p className="hero__subtitle">{siteConfig.tagline}</p>
+        <img src={heroImg} alt="TODO" className={styles.heroImg} />
```

Comment:
You can set the `alt` as `hero__image` to match the rest.

Reply:
Ah, forgot about the TODO. The alt should be describing the image, for accessibility purposes. Added in a description.